persistence-dse-6.8/pom.xml

```diff
@@ -126,6 +126,11 @@
     </dependency>
 
     <!-- 3rd party dependencies -->
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec</artifactId>
+      <version>4.1.78.1.dse</version>
```

Comment:
Yes, I believe this is correct as per:

https://github.com/datastax/release-notes/blob/master/DSE_6.8_Release_Notes.md

for DSE 6.8.29